packages/frontend-2/components/settings/user/Profile.vue

```diff
@@ -7,13 +7,32 @@
       <SettingsUserProfileChangePassword :user="user" />
       <hr class="my-6 md:my-10" />
       <SettingsUserProfileDeleteAccount :user="user" />
+      <hr class="my-6 md:my-10" />
+      <div class="text-xs text-foreground-2 w-full flex flex-col space-y-2">
+        <div class="flex">
+          User ID: #{{ user.id }}
+          <ClipboardIcon
+            class="w-4 h-4 ml-2 cursor-pointer hover:text-foreground transition"
+            @click="copyUserId"
+          />
+        </div>
+        <div v-if="distinctId" class="flex">
+          {{ distinctId }}
+          <ClipboardIcon
```

Comment:
We dont really have the right button for this but I'll adjust the styling of this anyway once we have the design system updates